octobot_evaluators/data_manager/matrix_manager.py

```diff
@@ -177,3 +196,35 @@ async def subscribe_nodes_event(matrix_id, nodes_path, callback, timeout=None):
     """
     await get_nodes_event(matrix_id, nodes_path, timeout=timeout)
     callback()
+
+
+def is_tentacle_value_valid(matrix_id, tentacle_path, timestamp=0, delta=10) -> bool:
+    """
+    # TODO This method only works with complete default path
```

Comment:
Should we keep this todo or add a try/except block?

Reply:
Yeah a try except is better. I don't know if we should let the TODO here. I don't know if we want to think about this issue later.